### PR TITLE
add zls 0.10.0

### PR DESCRIPTION
**Pull request by Slach**
zig LSP server

### CODE REVIEW

File: bucket/zls.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.10.0",
+    "description": "Zig language LSP server",
+    "homepage": "https://github.com/zigtools/zls",
+    "url": "https://install.zigtools.org/",
```
Comment (StarsbySea) [inline bucket/zls.json:5]:
```suggestion

```

**Review by StarsbySea — CHANGES_REQUESTED** [review]

**Comment by HUMORCE** [comment]
main bucket.

**Comment by Slach** [comment]
ok. create separate PR in MAIN bucket
https://github.com/ScoopInstaller/Main/pull/4886

**Pull request closed without merging**